src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -36,30 +36,15 @@ import useObservable from 'react-use/lib/useObservable';
 import { Observable } from 'rxjs';
 import { ChromeNavLink } from '../..';
 import { ChromeBranding } from '../../chrome_service';
-import { HttpStart } from '../../../http';
-
-function findClosestAnchor(element: HTMLElement): HTMLAnchorElement | void {
-  let current = element;
-  while (current) {
-    if (current.tagName === 'A') {
-      return current as HTMLAnchorElement;
-    }
-
-    if (!current.parentElement || current.parentElement === document.body) {
-      return undefined;
-    }
-
-    current = current.parentElement;
-  }
-}
+import { Logos } from '../../../../common/types';
```

Comment:
I think this is a new pattern you intentionally introduced to keep types out of the default exports, but we probably also need to update our guidance which says that we should import only from top level modules (`../../../../common`), in this case: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#import-only-top-level-modules

Reply:
oh! i will rexport them as `type` from the top level until we can talk about departure from that.